src/util/3d-tilt-hover-effect.ts

```diff
@@ -1,151 +1,143 @@
 /**
  * Utility functions for creating a 3D tilt hover effect.
  *
- * This module provides functions that enables consumer components to display a nice 3D effect,
- * when being hovered; enabling them to follow the cursor's position and tilt towards it.
+ * This module provides functions to enable a 3D tilt effect for consumer components,
+ * allowing elements to visually follow the cursor's position and tilt towards it.
+ * It also includes a glow effect for added interactivity.
  *
- * ## What you need, to make this work
- * ### Typescript
- * 1. Import the functions:
+ * ## Usage
+ *
+ * 1. **Import the utility**:
  *
  * ```tsx
- * import {
- *     tiltFollowingTheCursor,
- *     handleMouseEnter,
- *     handleMouseLeave,
- * } from './path/to/3d-tilt-hover-effect';
+ * import { getMouseEventHandlers } from './path/to/3d-tilt-hover-effect';
  * ```
  *
- * 2. In your component, define the necessary properties:
+ * 2. **Initialize in your component**:
+ *
+ * Call `getMouseEventHandlers()` in your component to retrieve the `handleMouseEnter`
+ * and `handleMouseLeave` functions, and attach them to the target element.
+ * For example, if your interactive element is the host itself:
```

Comment:
> if your interactive element is the host itself:

The interactive element can be the `host`, but its parent must have these styles:
```scss
@mixin parent-of-the-3d-element {
    isolation: isolate;
    transform-style: preserve-3d;

    perspective: 1000px;
    @media (prefers-reduced-motion) {
        perspective: 2000px;
    }
}
```

For the 3D effect to appear properly. For components like Card or Info tile, the only way is to use a nested element, and use it as the interactive element. Not sure if it's enough that we mention which mixins they should use, or if we should document this further.